generator/README.md

```diff
@@ -17,10 +17,12 @@ Once [Java v1.8](https://www.java.com/en/download/manual.jsp) is installed you c
 
 `java -jar <path to JAR file> <command> [options] <arguments>`
 
-* `<command>` one of the commands described below, `generate`, `visualise`, and so on
+* `<command>` one of the commands described below, `generate`, `visualise` etc which are **case sensitive**
 * `[options]` optionally a combination of options to configure how the command operates
 * `<arguments>` required inputs for the command to operate
 
+**Note:** when using quoted `<arguments>` a trailing backslash will result in an error
```

Comment:
Can this be rephrased as something like: "don't include a trailing \ in directory paths."

We know it creates an error, but the above says what to do rather than what will happen.